lib/ts/recipe/emailpassword/types.ts

```diff
@@ -198,7 +198,9 @@ export type ResetPasswordUsingTokenUserInput = {
     /*
      * enterEmailForm config.
      */
-    enterEmailForm?: FeatureBaseConfig;
+    enterEmailForm?: FeatureBaseConfig & {
+        getDefaultValue?: () => string;
```

Comment:
no.. this does not go here. It should already be a part of the formField config. Why have you added this here?

Reply:
this is for user input 
formField was never part of the emailForm Input
It gets normalized into formField type later for resetPassword